### DISCUSSION TITLE
Welcome to pylbm Discussions!

### DISCUSSION THREAD

**Discussion by gouarin** [pylbm/pylbm#126] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ryousofvand** [comment]
I am working on fluid flow simulation in human body using a 2D MPI parallel LBM python code that I have developed. I am considering simulating 3D problems with larger domain sizes, so I was looking at the pylbm package and it looks great! I have a couple of questions;
About MPI support; is it dividing the physical domain to the number of assigned cores and taking care of communication between domains? 
Also, do you have any plane to add multi phase and fluid structure interaction to the package?  
Thank you!

**Reply by gouarin** [reply]
Hi @ryousofvand ,
about MPI, your domain is split according to the number of processes and pylbm makes the communication between the subdomains for you.
We plan to add multi phase examples but not fluid-structure interaction for now.
